pkg/agent/cniserver/pod_configuration.go

```diff
@@ -111,7 +112,8 @@ func parseContainerIPs(ipcs []*current.IPConfig) ([]net.IP, error) {
 func buildContainerConfig(
 	interfaceName, containerID, podName, podNamespace string,
 	containerIface *current.Interface,
-	ips []*current.IPConfig) *interfacestore.InterfaceConfig {
+	ips []*current.IPConfig,
+	vlanID *uint16) *interfacestore.InterfaceConfig {
```

Comment:
Same here.

Reply:
Refactored all vlanID from `*uint16` to `uint16`